eng/Version.Details.xml

```diff
@@ -1,33 +1,33 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.DotNet.Cli.Runtime" Version="3.0.100-preview.19104.2">
+    <Dependency Name="Microsoft.DotNet.Cli.Runtime" Version="3.0.100-preview3.19113.3">
       <Uri>https://github.com/dotnet/cli</Uri>
-      <Sha>833970dd00410700bd1b688d2addc1ba74101a7d</Sha>
+      <Sha>1f0cca90aa3ac87431451dc00c12dde4f9456479</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NET.Sdk" Version="3.0.100-preview.19104.2">
+    <Dependency Name="Microsoft.NET.Sdk" Version="3.0.100-preview3.19112.2">
       <Uri>https://github.com/dotnet/sdk</Uri>
-      <Sha>49a6842940a884f2a06860fa9173c2f11963618d</Sha>
+      <Sha>d9728e9670ae1084001be8d3d74f33c51e828c87</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Build" Version="16.1.0-preview.4">
+    <Dependency Name="Microsoft.Build" Version="16.0.0-preview.442">
```

Comment:
Is this a downgrade?

Reply:
It is. I can see how that would happen in a merge from 3.0 to master if their subscribed to different msbuild branches as most repos are.

Reply:
Just wanted to call it out so we resolve the conflict correctly, thanks!

Reply:
(sorry)

Reply:
It's my fault for asking a rhetorical question.  I should have made my intention more clear; thus, I'm the one who's sorry 😄 .